### PR TITLE
Migrate all the historical votes!

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

File: packages/lesswrong/server/migrations/2020-05-06-forceSecureImageLinks.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ registerMigration({
   action: async () => {
     await forEachDocumentBatchInCollection({
       collection: Posts,
+      batchSize: 500,
```
Comment (Raemon) [inline packages/lesswrong/server/migrations/2020-05-06-forceSecureImageLinks.ts:92]: doublechecking this file meant to get changed?

Comment (Discordius) [reply packages/lesswrong/server/migrations/2020-05-06-forceSecureImageLinks.ts:92]: Yeah, it's a small bugfix that seemed nice to have. I also want us to run this migration soon, so I was experimenting with it while I was doing a bunch of other migration-related work.

**Review by jimrandomh — APPROVED** [review]

**Comment by jimrandomh** [comment]
@Discordius Status update on this PR? It's been in "reviewed" state, waiting to merge, for over a month.

**Comment by jimrandomh** [comment]
@Discordius Ping again on this one.

**Pull request merged**